lib/galaxy/jobs/runners/__init__.py

```diff
@@ -169,9 +169,9 @@ def prepare_job(self, job_wrapper, include_metadata=False, include_work_dir_outp
                 include_metadata=include_metadata,
                 include_work_dir_outputs=include_work_dir_outputs,
             )
-        except:
+        except Exception, e:
```

Comment:
You don't need to change this PR, but we should move toward Python 3 exceptions going forward ``except Exception as e:``